alert-database/src/test/java/com/synopsys/integration/alert/database/api/DefaultNotificationManagerTest.java

```diff
@@ -47,8 +48,8 @@
 
     @Test
     public void saveAllNotificationsTest() throws Exception {
-        Date createdAt = new Date();
-        Date providerCreationTime = new Date(System.currentTimeMillis() - 10000);
+        OffsetDateTime createdAt = new Date();
+        OffsetDateTime providerCreationTime = new Date(System.currentTimeMillis() - 10000);
```

Comment:
Does this work? I don't see any methods that accept a date object in OffsetDateTime, would this need to be rewritten to something like `OffsetDateTime.now()`?

Reply:
Good catch.